ext/FluxnetSimulations.jl

```diff
@@ -0,0 +1,308 @@
+module FluxnetSimulations
+import ClimaUtilities.TimeVaryingInputs:
+    TimeVaryingInput, LinearInterpolation, PeriodicCalendar
+import ClimaUtilities.TimeManager: ITime, date
+using Thermodynamics
+using Dates
+using DelimitedFiles
+using Format
+using DocStringExtensions
+using Insolation
+import ClimaLand.Parameters as LP
+using ClimaLand
+export prescribed_forcing_fluxnet, set_fluxnet_ic!
+"""
+     prescribed_forcing_fluxnet(site_ID,
+                                lat,
+                                long,
+                                hour_offset_from_UTC,
+                                start_date,
+                                earth_param_set,
+                                FT;
+                                gustiness=1,
+                                c_co2 = TimeVaryingInput((t) -> 4.2e-4),
+                                time_interpolation_method = LinearInterpolation(PeriodicCalendar())) # not used??
+
+A helper function which constructs the `PrescribedAtmosphere` and `PrescribedRadiativeFluxes`
+from a file path pointing to the Fluxnet data in a csv file, the start date, latitude, longitude,
+the hour offset of the site from UTC (local_time + offset = time in UTC),
+and the earth_param_set.
+"""
+function prescribed_forcing_fluxnet(
+    site_ID,
+    lat,
+    long,
+    hour_offset_from_UTC,
+    atmos_h,
+    start_date, # in UTC
+    earth_param_set,
+    FT;
+    split_precip = true,
+    gustiness = 1,
+    c_co2 = TimeVaryingInput((t) -> 4.2e-4),
+    time_interpolation_method = LinearInterpolation(PeriodicCalendar()),
+)
+    fluxnet_csv_path = ClimaLand.Artifacts.experiment_fluxnet_data_path(site_ID)
+    driver_data = readdlm(fluxnet_csv_path, ',')
+    columns = driver_data[1, :]
+    labels = ("TA_F", "VPD_F", "PA_F", "P_F", "WS_F", "LW_IN_F", "SW_IN_F")
+    indices = [findfirst(columns .== label) for label in labels]
+
+    if any(indices .== nothing)
+        nothing_id = findall(indices .== nothing)
+        @error("$(labels[nothing_id]) is missing in the data, but required.")
+    end
+
+    index_map = Dict(zip(labels, indices))
+    LOCAL_DATETIME = DateTime.(format.(driver_data[2:end, 1]), "yyyymmddHHMM")
+    UTC_DATETIME = LOCAL_DATETIME .+ Dates.Hour(hour_offset_from_UTC)
+    DATA_DT = Second(LOCAL_DATETIME[2] - LOCAL_DATETIME[1]).value # seconds
+    seconds_since_start_date =
+        [Second(UTC - start_date).value for UTC in UTC_DATETIME]
+
+    atmos_T_data = driver_data[2:end, index_map["TA_F"]]
```

Comment:
Can you add a note above this section that we don't care about missing data because it'll be handled by the TVI?

Reply:
Actually, can the TVI handle values of `Missing` in the data array, or do we have to filter them out like this?

Reply:
I actually need to double check. I think sometimes it is Missing, and sometimes missing data is marked by -9999 or something

TVI cannot handle missing data marked by Missing

Reply:
I added a comment in the corresponding location in the new code